### ISSUE TITLE
[Feature request] Morty Proxy only for images

### ISSUE THREAD

**Issue by sevospl** [searx/searx#1826]
It would be nice to be able to use Morty Proxy only for images.

**Comment by unixfox** [comment]
Probably better to open an issue on the dedicated github repository: https://github.com/asciimoo/morty/issues/new

**Comment by sevospl** [comment]
> Probably better to open an issue on the dedicated github repository: https://github.com/asciimoo/morty/issues/new

Not really, it needs to be done in the Searx code itself. Morty doesn't have control over how Searx works.
